docs/manual/document-format-reference/job-yaml-v12.md

```diff
@@ -186,13 +186,25 @@ value of `failed`. Allowed values:
 
 : Job schedule
 
+`nodeFilterEditable`
+
+: Boolean option that determines if Node filters can be edited when running the job. (true | false)
+
 [`nodefilters`](#nodefilters)
 
 : Node filter definition
 
+`nodesSelectedByDefault`
+
+: Boolean option that determines if Nodes (based on node filter) are selected automatically or if the user running the job must select nodes from the results. (true | false)
+
 [`notification`](#notification)
 
-: Job result notifications
+: Job result notifications.
+
+[`orchestration`](#orchestration)
```

Comment:
since the section is actually called `orchestrator` i think this and the header section later should be renamed to "Orchestrator"